src/exception/fail.cpp

```diff
@@ -0,0 +1,40 @@
+//
+// Created by Dawid Drozd aka Gelldur on 6/30/16.
+//
+
+#include "fail.hpp"
+
+#include <stdexcept>
+
+namespace commoncpp::exception::fail
+{
+
+std::string trimFromSrcFolder(std::string_view where)
+{
+	auto srcPosition = where.rfind("src/");
+	if(srcPosition != std::string::npos)
+	{
+		return std::string{where.substr(srcPosition + 4)};
```

Comment:
Looking at the code for the first time and I can't tell what 4 is in this case, constexpr it please.